src/bot/discord.js

```diff
@@ -33,37 +35,21 @@ client.on('messageCreate', async msg => {
   }
 
   if (action === '!drip') {
-    if (_.isEmpty(arg0)) {
-      msg.reply('please enter a wallet address after !drip.');
-      return;
-    }
-
-    try {
-      pdKeyring.decodeAddress(arg0);
-    } catch (e) {
-      msg.reply(`address ${arg0} entered is incompatible to ${networkName}.`);
-      return;
-    }
-
-    const res = await ax.post('/bot-endpoint', {
-      sender,
-      address: arg0,
-      amount: sendAmount * units,
-    });
-
-    if (res.data === 'LIMIT') {
-      msg.reply(`your Discord ID or the address has reached its daily quota. Please request only once every 24 hours.`);
-      return;
-    }
+    msg.reply(await actions.drip(sender, args[1]));
+    return;
+  }
 
-    msg.reply(`I just sent ${sendAmount} ${tokenSymbol} to address ${arg0}. Extrinsic hash: ${res.data}.`);
+  if (action === '!drip-later') {
+    msg.reply(await actions.dripLater(sender, args[1], _.join(_.slice(args, 2), ' ')));
+    return;
   }
 
   if (action === '!faucet') {
     msg.reply(`
 Usage:
   !balance - Get the faucet's balance.
   !drip <Address> - Send ${tokenSymbol}s to <Address>.
+  !drip-later <Address> <Time> - Send ${tokenSymbol}s to <Address> later. Time format(UTC): 4/14 at 5pm
```

Comment:
Let's change the input format to be more international friendly, let's do `04-14-2022 at 5PM`

Reply:
Fixed